<ui version="4.0" >
 <class>KWin::WaterEffectConfigForm</class>
 <widget class="QWidget" name="KWin::WaterEffectConfigForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="2" >
    <widget class="QSpinBox" name="gridSizeSpin" >
     <property name="minimum" >
      <number>1</number>
     </property>
     <property name="maximum" >
      <number>50</number>
     </property>
     <property name="value" >
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3" >
    <widget class="QCheckBox" name="wavesOnMouseBox" >
     <property name="text" >
      <string>Make waves with Ctrl + Left mouse button</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="gridSize" >
     <property name="text" >
      <string>&amp;Grid size:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy" >
      <cstring>gridSizeSlider</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QSlider" name="gridSizeSlider" >
     <property name="minimum" >
      <number>1</number>
     </property>
     <property name="maximum" >
      <number>50</number>
     </property>
     <property name="value" >
      <number>10</number>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3" >
    <widget class="QCheckBox" name="wavesOnActivationBox" >
     <property name="text" >
      <string>Make waves on window activation</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Default wave</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label_6" >
        <property name="text" >
         <string>&amp;Amplitude:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy" >
         <cstring>amplitudeSlider</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QSlider" name="amplitudeSlider" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum" >
         <number>5</number>
        </property>
        <property name="maximum" >
         <number>200</number>
        </property>
        <property name="value" >
         <number>40</number>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QSpinBox" name="amplitudeSpin" >
        <property name="maximumSize" >
         <size>
          <width>67</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="baseSize" >
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="layoutDirection" >
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="alignment" >
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="buttonSymbols" >
         <enum>QAbstractSpinBox::UpDownArrows</enum>
        </property>
        <property name="minimum" >
         <number>5</number>
        </property>
        <property name="maximum" >
         <number>300</number>
        </property>
        <property name="value" >
         <number>40</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_5" >
        <property name="text" >
         <string>&amp;Periode length:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy" >
         <cstring>periodeSlider</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QSlider" name="periodeSlider" >
        <property name="minimum" >
         <number>20</number>
        </property>
        <property name="maximum" >
         <number>200</number>
        </property>
        <property name="value" >
         <number>80</number>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QSpinBox" name="periodeSpin" >
        <property name="maximumSize" >
         <size>
          <width>67</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="baseSize" >
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="layoutDirection" >
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="alignment" >
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="buttonSymbols" >
         <enum>QAbstractSpinBox::UpDownArrows</enum>
        </property>
        <property name="minimum" >
         <number>20</number>
        </property>
        <property name="maximum" >
         <number>200</number>
        </property>
        <property name="value" >
         <number>80</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_4" >
        <property name="text" >
         <string>&amp;Speed:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy" >
         <cstring>speedSlider</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QSlider" name="speedSlider" >
        <property name="minimum" >
         <number>10</number>
        </property>
        <property name="maximum" >
         <number>500</number>
        </property>
        <property name="singleStep" >
         <number>0</number>
        </property>
        <property name="value" >
         <number>250</number>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="2" >
       <widget class="QSpinBox" name="speedSpin" >
        <property name="maximumSize" >
         <size>
          <width>67</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="baseSize" >
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="layoutDirection" >
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="alignment" >
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="buttonSymbols" >
         <enum>QAbstractSpinBox::UpDownArrows</enum>
        </property>
        <property name="minimum" >
         <number>10</number>
        </property>
        <property name="maximum" >
         <number>500</number>
        </property>
        <property name="singleStep" >
         <number>1</number>
        </property>
        <property name="value" >
         <number>250</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="label_7" >
        <property name="text" >
         <string>&amp;Damping start:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy" >
         <cstring>dampingSlider</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QSlider" name="dampingSlider" >
        <property name="minimum" >
         <number>5</number>
        </property>
        <property name="maximum" >
         <number>2000</number>
        </property>
        <property name="value" >
         <number>100</number>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="2" >
       <widget class="QSpinBox" name="dampingSpin" >
        <property name="maximumSize" >
         <size>
          <width>67</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="baseSize" >
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="layoutDirection" >
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="alignment" >
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="buttonSymbols" >
         <enum>QAbstractSpinBox::UpDownArrows</enum>
        </property>
        <property name="minimum" >
         <number>5</number>
        </property>
        <property name="maximum" >
         <number>2000</number>
        </property>
        <property name="value" >
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0" >
       <widget class="QLabel" name="label_8" >
        <property name="text" >
         <string>&amp;Radius:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy" >
         <cstring>radiusSlider</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1" >
       <widget class="QSlider" name="radiusSlider" >
        <property name="minimum" >
         <number>5</number>
        </property>
        <property name="maximum" >
         <number>2000</number>
        </property>
        <property name="value" >
         <number>250</number>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="2" >
       <widget class="QSpinBox" name="radiusSpin" >
        <property name="maximumSize" >
         <size>
          <width>67</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="baseSize" >
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="layoutDirection" >
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="alignment" >
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="buttonSymbols" >
         <enum>QAbstractSpinBox::UpDownArrows</enum>
        </property>
        <property name="minimum" >
         <number>5</number>
        </property>
        <property name="maximum" >
         <number>2000</number>
        </property>
        <property name="value" >
         <number>250</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0" colspan="3" >
    <widget class="QGroupBox" name="groupBox_8" >
     <property name="title" >
      <string>Activation</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3" >
      <item row="0" column="0" colspan="2" >
       <widget class="KWin::GlobalShortcutsEditor" native="1" name="editor" >
        <property name="minimumSize" >
         <size>
          <width>0</width>
          <height>200</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="3" >
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Rain</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4" >
      <item row="0" column="2" >
       <widget class="QSpinBox" name="rainDelaySpin" >
        <property name="maximumSize" >
         <size>
          <width>67</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="baseSize" >
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="layoutDirection" >
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="alignment" >
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="buttonSymbols" >
         <enum>QAbstractSpinBox::UpDownArrows</enum>
        </property>
        <property name="minimum" >
         <number>10</number>
        </property>
        <property name="maximum" >
         <number>1000</number>
        </property>
        <property name="value" >
         <number>30</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QSlider" name="rainDelaySlider" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum" >
         <number>10</number>
        </property>
        <property name="maximum" >
         <number>1000</number>
        </property>
        <property name="value" >
         <number>30</number>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="label_9" >
        <property name="text" >
         <string>Rain delay:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy" >
         <cstring>rainDelaySlider</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KWin::GlobalShortcutsEditor</class>
   <extends>QWidget</extends>
   <header location="global" >kwineffects.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>gridSizeSlider</tabstop>
  <tabstop>gridSizeSpin</tabstop>
  <tabstop>wavesOnActivationBox</tabstop>
  <tabstop>wavesOnMouseBox</tabstop>
  <tabstop>amplitudeSlider</tabstop>
  <tabstop>amplitudeSpin</tabstop>
  <tabstop>periodeSlider</tabstop>
  <tabstop>periodeSpin</tabstop>
  <tabstop>speedSlider</tabstop>
  <tabstop>speedSpin</tabstop>
  <tabstop>dampingSlider</tabstop>
  <tabstop>dampingSpin</tabstop>
  <tabstop>radiusSlider</tabstop>
  <tabstop>radiusSpin</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>amplitudeSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>amplitudeSpin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>151</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel" >
     <x>395</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>amplitudeSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>amplitudeSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>401</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel" >
     <x>193</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>periodeSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>periodeSpin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>178</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel" >
     <x>401</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>periodeSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>periodeSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>409</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel" >
     <x>177</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>speedSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>speedSpin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>152</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel" >
     <x>399</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>speedSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>speedSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>402</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel" >
     <x>179</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dampingSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>dampingSpin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>170</x>
     <y>215</y>
    </hint>
    <hint type="destinationlabel" >
     <x>397</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dampingSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>dampingSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>417</x>
     <y>216</y>
    </hint>
    <hint type="destinationlabel" >
     <x>184</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radiusSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>radiusSpin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>169</x>
     <y>246</y>
    </hint>
    <hint type="destinationlabel" >
     <x>393</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radiusSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>radiusSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>409</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel" >
     <x>173</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gridSizeSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>gridSizeSpin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>108</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel" >
     <x>396</x>
     <y>21</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gridSizeSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>gridSizeSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>403</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel" >
     <x>197</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rainDelaySlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>rainDelaySpin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>218</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel" >
     <x>397</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rainDelaySpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>rainDelaySlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>381</x>
     <y>313</y>
    </hint>
    <hint type="destinationlabel" >
     <x>267</x>
     <y>313</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
